cmd/vulcan-exposed-router-ports/main.go

```diff
@@ -61,44 +62,36 @@ var (
 	}
 )
 
-func exposedRouterPorts(target string, nmapReport *gonmap.NmapRun) []report.Vulnerability {
-	gr := report.ResourcesGroup{
-		Name: "Network Resources",
-		Header: []string{
-			"Hostname",
-			"Port",
-			"Protocol",
-			"Service",
-			"Version",
-		},
-	}
-
-	add := false
+func exposedRouterPorts(target string, nmapReport *gonmap.NmapRun, state checkstate.State) {
 	for _, host := range nmapReport.Hosts {
 		for _, port := range host.Ports {
 			if port.State.State != "open" {
 				continue
 			}
-
-			add = true
-
-			networkResource := map[string]string{
-				"Hostname": target,
-				"Port":     strconv.Itoa(port.PortId),
-				"Protocol": port.Protocol,
-				"Service":  port.Service.Product,
-				"Version":  port.Service.Version,
-			}
-			gr.Rows = append(gr.Rows, networkResource)
+			vuln := exposedVuln
+			vuln.AffectedResource = fmt.Sprintf("%d/%s", port.PortId, port.Protocol)
+			vuln.Fingerprint = helpers.ComputeFingerprint(port.Protocol, port.Service)
+			vuln.Labels = []string{"issue", port.Protocol, "discovery"}
```

Comment:
I'd avoid adding dynamic labels for now, as the current approach is that labels are stored at _Issue_ level in the Vulnerability DB. There are ongoing discussions about it but we don't know yet what will be the outcome of those.

Reply:
Also, in this case the label should be `informational` instead of `issue`.

Reply:
Ok, so? just remove it?
And, is it `issue` or `informational`?

Reply:
ok